core/src/test/scala/unit/kafka/log/LogCleanerTest.scala

```diff
@@ -569,13 +572,13 @@ class LogCleanerTest {
     log.appendAsLeader(commitMarker(producerId, producerEpoch), leaderEpoch = 0, isFromClient = false)
     log.roll()
 
-    // delete horizon set to 0 to verify marker is not removed early
-    val dirtyOffset = cleaner.doClean(LogToClean(tp, log, 0L, log.activeSegment.baseOffset), deleteHorizonMs = 0L)._1
+    // delete horizon set to some smaller timestamp to verify marker is not removed early
```

Comment:
Could we change the comment to the following?

"Aborted records are removed, but the abort marker is still preserved."